content/blog/docker-containers/index.md

```diff
@@ -0,0 +1,93 @@
+The [Pulumi Docker Containers](https://github.com/pulumi/pulumi-docker-containers) provide a convenient way for running Pulumi in CI/CD pipelines, or for running Pulumi in environments where you don't want to install the Pulumi CLI directly. These images also power [Pulumi Deployments](/product/pulumi-deployments/). We provide several flavors of images, including the [pulumi/pulumi](https://hub.docker.com/r/pulumi/pulumi/tags) image that includes all supported language runtimes in a single image, as well as slim images for each language runtime, for example [pulumi/pulumi-python](https://hub.docker.com/r/pulumi/pulumi-python/tags) or [pulumi/pulumi-nodejs](https://hub.docker.com/r/pulumi/pulumi-nodejs/tags).
+
+To provide more flexibility for choosing the version of the programming language to use, we have added versioned images for the language specific images covering [all of our supported languages](https://github.com/pulumi/pulumi?tab=readme-ov-file#languages). We now also support setting the version of the Node.js and Python runtimes used in Pulumi Deployments by using `.node-version` and `.python-version` files.
+
+<!--more-->
+
+For Python we provide images for [Python 3.9 to 3.12](https://hub.docker.com/u/pulumi?page=1&search=pulumi-python), for Node.js we have images for [Node.js 18, 20 and 22](https://hub.docker.com/u/pulumi?page=1&search=pulumi-nodejs), and for .NET we have images for [.NET 6.0 and 8.0](https://hub.docker.com/u/pulumi?page=1&search=pulumi-dotnet). These versioned images include the language version in the image name, for example [pulumi/pulumi-python-3.12](https://hub.docker.com/r/pulumi/pulumi-python-3.12/tags) or [pulumi/pulumi-nodejs-22](https://hub.docker.com/r/pulumi/pulumi-nodejs-22/tags).
+
+{{% notes type="info" %}}
+The Go image is not versioned, as the Go runtime natively supports multiple versions via its [Go Toolchains](https://go.dev/doc/toolchain) feature. The `go` statement in your Pulumi project's `go.mod` file determines the Go version used to compile your program.
+{{% /notes %}}
+
+### Updating the Default Language Versions
+
+In **November 2024** the images without a version suffix, like [pulumi/pulumi-python](https://hub.docker.com/r/pulumi/pulumi-python/tags), and the [pulumi/pulumi](https://hub.docker.com/r/pulumi/pulumi) image, will be updated to use the latest versions for each of the language runtimes by default:
+
+| Language Runtime | Current Version | New Version |
+| ---------------- | --------------- | ----------- |
+| .NET             | 6.0             | 8.0         |
+| Go               | 1.21            | 1.23        |
+| Node.js          | 18              | 22          |
+| Python           | 3.9             | 3.13        |
+| Java             | 17              | 21          |
+
+The default versions will be updated as new versions of the language runtimes are released, matching the [versions supported by the Pulumi CLI](https://github.com/pulumi/pulumi?tab=readme-ov-file#languages).
```

Comment:
```suggestion
The default versions will be updated as new versions of the language runtimes are released, matching the [versions supported by the Pulumi CLI](https://github.com/pulumi/pulumi?tab=readme-ov-file#languages).  This will enable the unversioned [pulumi/pulumi](https://hub.docker.com/r/pulumi/pulumi/tags) image to stay evergreen as new runtimes come into and out of support.  
```